RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1131,6 +1136,52 @@ private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstrac
         return false;
     }
 
+    
+    /**
+     *  Finds the Host and Domain information from the registry.
+     */
+    private void parseSystemHostDomain() {
+        List<AbstractFile> regFiles = findRegistryFiles();
+
+        for (AbstractFile systemHive: regFiles) {
+            if (systemHive.getName().toLowerCase().equals("system")) {
+                
+                String systemFileNameLocal = RAImageIngestModule.getRATempPath(currentCase, "reg") + File.separator + systemHive.getName();
+                File systemFileNameLocalFile = new File(systemFileNameLocal);
+        
+                if (!systemFileNameLocalFile.exists()) {
+                    try {
+                        ContentUtils.writeToFile(systemHive, systemFileNameLocalFile, context::dataSourceIngestIsCancelled);
+                    } catch (ReadContentInputStreamException ex) {
+                        logger.log(Level.WARNING, String.format("Error reading registry file '%s' (id=%d).",
+                                systemHive.getName(), systemHive.getId()), ex); //NON-NLS
+                        this.addErrorMessage(
+                                NbBundle.getMessage(this.getClass(), "ExtractRegistry.analyzeRegFiles.errMsg.errWritingTemp",
+                                this.getName(), systemHive.getName()));
+                        continue;
+                    } catch (IOException ex) {
+                        logger.log(Level.SEVERE, String.format("Error writing temp registry file '%s' for registry file '%s' (id=%d).",
+                                systemFileNameLocal, systemHive.getName(), systemHive.getId()), ex); //NON-NLS
+                        this.addErrorMessage(
+                                NbBundle.getMessage(this.getClass(), "ExtractRegistry.analyzeRegFiles.errMsg.errWritingTemp",
+                                        this.getName(), systemHive.getName()));
+                        continue;
+                    }                
+                }
+					
+                try {
+                    ParseRegistryHive systemRegFile = new ParseRegistryHive(systemFileNameLocalFile);
+                    hostName = systemRegFile.getRegistryKeyValue("ControlSet001/Services/Tcpip/Parameters", "hostname");
+                    domainName = systemRegFile.getRegistryKeyValue("ControlSet001/Services/Tcpip/Parameters", "domain");
```

Comment:
Should you break out of the loop once you find this info?

Reply:
Yes, updated